README.md

```diff
@@ -1,3 +1,18 @@
 # Bibtex Online Converter
 
+## URL Params to Autopopulate fields
 
+| param  | Description                                                                           |
+| ------ | ------------------------------------------------------------------------------------- |
+| bibtex | bibtex input. Must be enclosed with quotation marks like: `"` &#124; `'` &#124; `     |
+| format | Can either be `apa` &#124; `mla` &#124; `chicago` &#124; `harvard` &#124; `vancouver` |
+
+### Example
+
+`?bibtex="@book{madigan1997brock,
+title={Brock biology of microorganisms},
+author={Madigan, Michael T and Martinko, John M and Parker, Jack and others},
+volume={11},
+year={1997},
+publisher={Prentice hall Upper Saddle River, NJ}
+}"?format=mla`
```

Comment:
Should be `&` instead of `?`

Reply:
Oh you're right! I didn't spot that. Thank you!